invenio_app_rdm/communities_ui/templates/semantic-ui/invenio_communities/collections/collection.html

```diff
@@ -0,0 +1,31 @@
+{# -*- coding: utf-8 -*-
+
+This file is part of Invenio.
+Copyright (C) 2024 CERN.
+
+Invenio is free software; you can redistribute it and/or modify it
+under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{% extends "invenio_communities/details/base.html" %}
+
+{%- block javascript %}
+{{ super() }}
+{{ webpack['invenio-app-rdm-collections-search.js'] }}
+{%- endblock %}
+
+{%- block page_body %}
+{{ super() }}
+<div class="ui container fluid theme-tertiary page-subheader rel p-10 mr-0-mobile ml-0-mobile">
```

Comment:
do we have any alternative for the collection name header? to me it looks too similar to the "Research and innovation" one

Reply:
Indeed, it's based on "Research and Innovation" so it can be improved.

Will have a look in a second PR, right now we want to move collections forward to have real data showing up.